src/Components/App/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ItemList from '../ItemList/ItemList';
+import Footer from '../Footer/footer';
+import InputItem from '../InputItem/InputItem';
+
+const todoItem = 'Create new app';
+const App = () => (
+<div> 
+	<h1> ToDO List </h1> 
+  	<InputItem/>
+  	<ItemList todoItem = {todoItem} />
+  	<Footer count = {3} />
```

Comment:
Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/gh/WebHeroSchool/todo_react_project/pullRequest?prid=5893382)